packages/orchestration/src/orchestration-completion-from-json.test.ts

```diff
@@ -0,0 +1,131 @@
+import { constructCompletionFromJson } from './orchestration-client.js';
+
+describe('constructCompletionFromJSON()', () => {
+  it('should accept a string orchestration config', () => {
+    const jsonConfig: string = `{
+            "module_configurations": {
+            "llm_module_config": {
+                "model_name": "gpt-35-turbo-16k",
+                "model_params": {
+                "max_tokens": 50,
+                "temperature": 0.1
+                }
+            },
+            "templating_module_config": {
+                "template": [{ "role": "user", "content": "Hello!" }]
+            }
+            }
+        }`;
+    const result: Record<string, any> = constructCompletionFromJson(jsonConfig);
+
+    expect(typeof jsonConfig).toBe('string');
+    expect(result.orchestration_config).toBeDefined();
+    expect(result.orchestration_config).toHaveProperty('module_configurations');
+  });
+
+  it('should throw an error when invalid JSON is provided', () => {
+    const invalidJsonConfig = '{ "module_configurations": {}, ';
+
+    expect(() => constructCompletionFromJson(invalidJsonConfig)).toThrow(
+      'Could not parse JSON'
+    );
+  });
+
+  it('with a JSON configuration and template params', () => {
```

Comment:
```suggestion
  it('should construct completion post request from JSON with input params', () => {
```

Reply:
[req] Prefer using natural language to describe a test case / collection.

Reply:
Yes I agree with this as well. We will also need to maintain this for rest of the unit tests in the project later. Because this is quite inconsistent atm. Here, I was just trying to make it consistent with `orchestration-completion-post-request.test.ts` but I also would prefer your suggested language.

Reply:
Oh, please ignore consistency and stick to the preferred natural language. (Or if not so much work, feel free to change that file as well)

@KavithaSiva Fyi.